lib/IRGen/GenProto.cpp

```diff
@@ -3734,7 +3752,7 @@ llvm::Value *irgen::emitWitnessTableRef(IRGenFunction &IGF,
 
   // In Embedded Swift, only class-bound wtables are allowed.
   if (srcType->getASTContext().LangOpts.hasFeature(Feature::Embedded)) {
```

Comment:
`IGF.IGM.Context` is cleaner IMO

Reply:
agreed. I will change it next time I work on this code